xwiki-platform-core/xwiki-platform-web/src/test/java/org/xwiki/web/DocumentTagsTest.java

```diff
@@ -112,7 +112,7 @@ void displayTagsWhenEditRightsAndTagPluginAvailableAndNoTags() throws Exception
         // - The tag label is displayed
         // - No tag is listed after the tag label
         // - The "+" link is displayed since the user has edit rights
-        assertThat(result, matchesPattern("\\Q<div class=\"doc-tags\" id=\"xdocTags\"> core.tags.list.label "
+        assertThat(result, matchesPattern("\\Q<div class=\"doc-tags \" id=\"xdocTags\"> core.tags.list.label "
```

Comment:
Why do say that it's good to have an extra space???

Reply:
You should not do stuff that make it easier for you when you don't understand something but instead do the right things! :)